lib/cast/cast.go

```diff
@@ -0,0 +1,74 @@
+// Package cast provides save casting functions for converting between types without panicking.
+package cast
```

Comment:
not sure what is the policy on testing but I feel this package should have a unit table test

Reply:
sure, I'll add

Reply:
done